plugins/modules/events.py

```diff
@@ -88,11 +88,15 @@
         ]
 """
 
-import os
 import traceback
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.basic import missing_required_lib
 
+try:
+    from ansible_collections.openshift_lab.assisted_installer.plugins.module_utils import apitoken
```

Comment:
Changes look good, is this a common issue, is this just the same module hosted in two different places? (from the ansible module vs from the openshift lab collection)?